code-challenges.test.js

```diff
@@ -13,8 +13,47 @@
 
 // --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
 
+/**
+ * - Return empty array for numbers less than 2
+ * - Initialize an empty array called "fib"
+ * - Assign the first two indexes to 0 and 1
+ * - Start the count at 2 and stop at the end marker we specifiy (e.g., "n")
+ * - Do the fibonacci algorithm and assign it to the fib array
+ * - Slice off the first index to remove 0 from the fib array
+ */
+
+var  fibonacci = (n) => {
+  if (n < 2) return [];
+
+  var fib = [];
+
+  fib[0] = 0;
+  fib[1] = 1;
+
+  for (let i = 2; i <= n; i++) {
+    fib[i] = fib[i - 2] + fib[i - 1];
+  }
+  return fib.slice(1);
+};
```

Comment:
The slice is interesting. I wonder if you could start your array with a 1 rather than 0 and avoid having to remove the first character.